src/logic/calculate.js

```diff
@@ -0,0 +1,117 @@
+import Big from 'big.js';
+
+export default function calculate(obj, buttonName) {
+  const { currentOperand, previousOperand, operation } = obj;
+  let result;
+
+  switch (buttonName) {
+    case 'AC':
+      return {
+        currentOperand: null,
+        previousOperand: null,
+        operation: null,
+      };
+    case '+/-':
+      if (currentOperand) {
+        result = Big(currentOperand).times(-1).toString();
+        return {
+          currentOperand: result,
+          previousOperand,
+          operation,
+        };
+      }
+      return obj;
```

Comment:
- [ ] Great effort working on the changes requested by the previous reviewer, but some of the changes are left undone. I would kindly suggest you change the `calculate.js` file too as the line of codes you have there looks different from the one given to us by Microverse to use for this milestone. I would advice we stick to the project requirement as much as possible. Kindly replace the line of codes we have in your `calculate.js` with what we have in the file given to us by Microverse.

Below is the link to the file 👇 

[Link](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js)

Reply:
Hi, can you see my project now? I haven't changed anything in calculate.js and operate.js files, but when I do calculations equals button doesn't work properly. I couldn't fix it. Can you help me, please?